### DISCUSSION TITLE
Does ISIC Scholar work for GitHub Education?

### DISCUSSION THREAD

**Discussion by kitchencel** [community/community#139927] [GitHub Education] [answered]
### Hi everyone,

So i am still in primary school (American Academy in Prague), and I was thinking of getting an ISIC Scholar for the GitHub benefits. I know the ISIC provides student benefits, but I am not sure about ISIC Scholar. Thanks in advance! ^^

-Vova

**Comment by davevad93** [comment, marked as answer]
Hi @vodaman ,

The requirements to be eligible for GitHub Education are the following, as indicated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student#requirements):

***To be eligible for GitHub Education, including GitHub Student Developer Pack and other benefits, you must:***

- ***Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution***
- ***Have a verifiable school-issued email address or upload documents that prove your current student status***
- ***Have a [GitHub personal account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github)***
- ***Be at least 13 years old***

***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***

If you're still in primary school, I don't think you can apply for GitHub Education yet.

**Reply by kitchencel** [reply]
oh. thanks! :)

**Reply by davevad93** [reply]
You're welcome @vodaman 👍
